config-tests/test_email_notification.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import requests
+from random import randint
+
+
+@pytest.mark.settings
+def test_email_notifications_work(env, conf):
+    if env != 'stage':
+        pytest.skip('Test skipped in production')
+
+    url = conf.get(env, 'qa_collection_url')
+    auth = (conf.get(env, 'qa_collection_login'), conf.get(env, 'qa_collection_password'))
+
+    # Add record to QA collection
+    data = {'data': {'qa': randint(1, 99999)}}
+    resp = requests.post(
+        json=data,
+        auth=auth,
+        url=url + '/records'
+    )
+    assert resp.status_code == 201
+
+    # Ask for a review
+    data = {'data': {'status': 'to-review'}}
+    resp = requests.patch(
+        json=data,
+        auth=auth,
+        url=url
+    )
+    assert resp.status_code == 200
+
+    # Look for the email that has the message that a review has been requested
+    resp = requests.get(url="https://restmail.net/mail/kintoemailer")
```

Comment:
Idea: you could add a safety check at the beginning to verify that `kintoemailer` is among the configured recipients

Reply:
Is there a way to determine that?

Reply:
The recipients are defined here:

https://github.com/mozilla-services/cloudops-deployment/blob/8e7be0316e1a49ff126604e7d714b6c209c119a6/projects/kinto/puppet/yaml/permissions/kinto.stage.yaml#L990-L995
(in the `/buckets/staging` endpoint, readable  for the members of the `observers` group)

As you can see, the recipients are currently the members of the `qa-reviewers` and `observers` groups :)

Reply:
I don't see where 'kintoemailer@restmail.com' is in that list of reviewers or observers. Is that value hard-coded somewhere?

Reply:
Me neither!

AFAIK there is nothing hard-coded on kinto side :|

Reply:
Ideally the test would use a unique restmail address, or the content of the email would be uniquely identifiable. If this test was run twice in succession, the second one could pass because the email from the first one is found.